packages/builders/scripts/inter-protocol/replace-electorate-core.js

```diff
@@ -55,13 +55,13 @@ const configurations = {
   },
   DEVNET: {
     committeeName: 'Economic Committee',
-    // TODO: Update the addresses after confirmation
     voterAddresses: {
       gov1: 'agoric1ldmtatp24qlllgxmrsjzcpe20fvlkp448zcuce',
       gov2: 'agoric140dmkrz2e42ergjj7gyvejhzmjzurvqeq82ang',
+      gov3: 'agoric1f0h5zgxyg3euxsqzs0506uj4cmu56y30pqx46s',
```

Comment:
That's gov4, isn't it?

Reply:
Yes, `gov3` is actually `gov4` in my sequential numbering. Sorry for any confusion! Do you suggest naming it `gov4`?

Reply:
yes, please!

Reply:
Done